File: app/src/test/java/com/mgaetan89/showsrage/fragment/SettingsFragmentTest.java
package com.mgaetan89.showsrage.fragment;

import com.mgaetan89.showsrage.R;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class SettingsFragmentTest {
	@Test
	public void getTitleResourceId() {
		assertThat(new SettingsFragment().getTitleResourceId()).isEqualTo(R.string.settings);
	}

	@Test
	public void getXmlResourceFile() {
		assertThat(new SettingsFragment().getXmlResourceFile()).isEqualTo(R.xml.settings);
	}
}

File: app/src/test/kotlin/com/mgaetan89/showsrage/fragment/SettingsFragmentTest.kt
package com.mgaetan89.showsrage.fragment

import com.mgaetan89.showsrage.R
import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class SettingsFragmentTest {
	@Test
	fun getTitleResourceId() {
		assertThat(SettingsFragment.newInstance().getTitleResourceId()).isEqualTo(R.string.settings)
	}

	@Test
	fun getXmlResourceFile() {
		assertThat(SettingsFragment.newInstance().getXmlResourceFile()).isEqualTo(R.xml.settings)
	}
}
